src/main/java/kostka/moviecatalog/configuration/DateConverter.java

```diff
@@ -0,0 +1,20 @@
+package kostka.moviecatalog.configuration;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+import java.time.LocalDate;
+import java.sql.Date;
+
+@Converter(autoApply = true)
+public class DateConverter implements AttributeConverter<LocalDate, Date> {
```

Comment:
Is is possible to use LocalDate on both sides? https://jdbc.postgresql.org/documentation/head/8-date-time.html

Reply:
in mysql database it is not possible afaik